avm/res/hybrid-compute/machine/tests/e2e/max.hci/main.test.bicep

```diff
@@ -0,0 +1,92 @@
+targetScope = 'subscription'
+
+metadata name = 'Creates an Arc Machine with maximum configurations'
+metadata description = 'This instance deploys the module with the full set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Required. The kind of machine to deploy.')
+param kind string = 'HCI'
+
```

Comment:
```suggestion
```

Should be coded into the test so that it also shows up in the readme example. Will comment a suggestion below